zipline/pipeline/engine.py

```diff
@@ -569,7 +574,14 @@ def _inputs_for_term(term, workspace, graph, domain):
                 out.append(input_data)
         return out
 
-    def compute_chunk(self, graph, dates, sids, initial_workspace, hooks):
+    def compute_chunk(self,
```

Comment:
We don't use `compute_chunk` downstream afaik, but we do have some tests that use it. Fixing that is tricky because I need the `start_date` and `end_date` of the chunk to enter the `computing_chunk` context manager, but the interface for `compute_chunk` is just that it runs over a range of "dates", which is actually the date range of the longest lookback window (that interface is a little wonky...)

I'm going to merge this to get the bugfix out. I'm not sure what the right fix is to get the internal engine interfaces cleaned up, but it's bigger than I want to try to tackle here.